src/private/encode.re

```diff
@@ -8,7 +8,7 @@ external float : float => Js.Json.t = "%identity"; /* [float n] makes a JSON num
 external int : int => Js.Json.t = "%identity"; /* [int n] makes a JSON number of the [int] [n] */
 
 /** [boolean b] makes a JSON boolean of the [Js.boolean] [b] */
-external boolean : Js.boolean => Js.Json.t = "%identity"; /* [boolean b] makes a JSON boolean of the [Js.boolean] [b] */
+external boolean : bool => Js.Json.t = "%identity"; /* [boolean b] makes a JSON boolean of the [Js.boolean] [b] */
```

Comment:
Not sure if the comment is still ok

Reply:
It should be [boolean b] makes a JSON boolean of the [bool] [b]